test_script.py

```diff
@@ -0,0 +1,4 @@
+# WARNING: THIS TEST IS MEANT TO DO A LIVE TEST OF THE CLIENT AND TESTS AGAINST A REAL FILE AND AGAINST A REAL WEBSITE
+from client.client import VariantAnnotationClient
+c = VariantAnnotationClient()
+print(c.annotate_file("variants.txt"))
```

Comment:
This script is what i used to test that this works against the real website and is provided to the interviewer to verify.